ncm-dirperm/src/main/perl/dirperm.pm

```diff
@@ -189,6 +211,40 @@ sub process_path {
 				return 0;
 			}
 		} elsif ($type eq "d") {
+            # Should we check if the needed mount is actually mounted?
+            # (i.e. don't build path on / (root) if the mount point is not available)
+            if ($pathentry->{checkmount}) {
+                # Which mountpoint is path built on?
+                my $path_mountpoint = undef;
```

Comment:
because you don not actually care about the vale of the moutnpotin, you can do
```
my @path_mountpoints = grep {$path =~ m/^$_\//} @configured_mounts;
if (@path_mountpoints) {
  # found mountpoint $path_mountpoints[0] is the one you are looking for 
} else {
  error()...
}
```